### ISSUE TITLE
 Called async_setup_platforms instead of awaiting async_forward_entry_setups

### ISSUE THREAD

**Issue by kuges** [markhannon/zimi-integration#4]
Getting the following warning in the HA log:

```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:77
First occurred: 11:05:50 AM (1 occurrences)
Last logged: 11:05:50 AM

Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for zimi using this method at custom_components/zimi/controller.py, line 94: self.hass.config_entries.async_setup_platforms(self.config, PLATFORMS)
```

**Comment by markhannon** [comment]
Please check version 1.1 of the integration

**Comment by markhannon** [comment]
Fixed in version 1.1
